packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -376,6 +376,13 @@ function parseBinding(
       result: { value: bindable },
     };
   }
+
+  if (bindingType === 'env') {
+    return {
+      scopePath,
+      result: { value: bindable },
```

Comment:
Could we make this return a new 
```
return {
      scopePath,
      envVarName: <% (name of environment variable) %>,
    };
```

and then make it so that the final arguments to `execBase` are the same as right now?

Reply:
Did you try this? Any outcome?